src/core/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@angular2-material/core",
+  "version": "2.0.0-alpha.0",
+  "description": "Angular 2 Material core",
+  "main": "./core.js",
+  "typings": "./core.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/angular/material2.git"
+  },
+  "keywords": [
+    "angular",
+    "material",
+    "material design",
+    "components"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/angular/material2/issues"
+  },
+  "homepage": "https://github.com/angular/material2#readme",
+  "peerDependencies": {
+    "angular2": "2.0.0-beta.8"
```

Comment:
can you use ^2.0.0-beta.8 instead?

I think it's better right now to error on the side of claiming compatibility with future Angular versions containing breaking changes rather than being too strict. If you are too strict people will not be able to use angular2@2.0.0-beta.9 until you cut a release. If you use ^ and we break material, then people can stick to beta.8 via package.json or shrinkwrap.